internal/local/command_service.go

```diff
@@ -252,6 +252,10 @@ func (c *command) runServiceVersionCommand(cmd *cobra.Command, _ []string) error
 }
 
 func (c *command) startService(service, configFile string) error {
+	if err := c.checkJavaVersion(service); err != nil {
```

Comment:
Is there something within `isRunning()` that requires Java? Otherwise I don't see how this changes anything.

Reply:
if user already has service running,`isRunning` will return and it won't get to validate Java version. That brought some confusion in that original ticket like "If Java version was not valid then why were some services started"? checking java version first will avoid such confusion. 
It's up to us how we want users to perceive the message.

Reply:
Wait, how can `isRunning` be true if the user doesn't have Java installed? Did the user start `confluent local` and then remove their Java installation?

Reply:
😕 this happened in that ticket. suppose a few services has been running before, and user changed java path / version and ran `start` again